.github/CODEOWNERS

```diff
@@ -68,6 +68,9 @@
 /fe/fe-core/src/main/java/com/starrocks/sql/optimizer/ @StarRocks/optimizer
 /fe/fe-core/src/main/java/com/starrocks/statistic/ @StarRocks/optimizer
 
+#query-scheduler
+/fe/fe-core/src/main/java/com/starrocks/qe/scheduler/ @StarRocks/query-scheduler
```

Comment:
All owners should be maintainers, how about renaming query-scheduler to scheduler-maintainer